m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+
+1. Open the ``package.json`` file and change the name of the package (remember to keep the content type name at the beginning, i.e. ``draganddrop-<name-of-your-acos-content-package>)``.
+2. Change the meta-information declared in the ``index.coffee`` file (that includes name, description, author, version and some other information that you consider relevant).
+3. Change the namespace of the package declared in the ``index.coffee`` file (again, remember to keep the content type name at the beginning, i.e. ``draganddrop-<name-of-your-acos-content-package>``.
+4. Change the name of the module from ``DragandDropExample`` to whichever name you prefer for your exercise (e.g. ``DragandDropDataManagement``). 
+
+After making all the changes mentioned above, your ``coffee.index`` should looks something like :ref:`the image below <acos-changes>`. Notice that in this example we did not change the meta-information, but you should change it.
+
+.. figure:: /images/gallery/index.coffee.png
+  :name: acos-changes
+  :width: 95%
+  :align: center
+
+  Diff view between the original file (left) and the edited one(right).
+
+Now that you have edited the some of the source files, you can proceed to install the ``coffeescript`` module and compile the ``index.coffee`` file. Therefore, you should run the following command in your project directory.
+
+.. code-block:: bash
+
+  npm install
+
+Verify that a ``node_modules`` folder and the ``index.js`` file are created. If the ``index.js`` file is not created, you should run the following command in your project directory.
+
+.. code-block:: bash
+
+  npm run prepare
+
+Create your own Acos exercise
+.............................
+Now that your project has been set up, it is time to start creating your own exercise. In this chapter, we will create a data management exercise so that you can follow step-by-step the process of creating Acos exercise. Remember that all the exercises you want to create should be located inside the ``exercises`` folder. You can also group them and organise them in subfolders. For example, in the current ``exercises`` folder, there is an ``images`` folder and an ``articles`` folder containing specific exercises. 
+
+For now, let's delete all the current examples and the related static files. Once, the ``exercises`` folder is empty we will start creating a new **XML** and **JSON**. In this case, we will call the exercise ``data_structure``. Therefore, we will have two files. One called ``data_structure.xml`` and the second one called ``data_structure.json``. Remember, that the name of those files should not contain hyphens and it is preferred to use an underscore to separate words in the name of the files. Finally, we will add our images to the ``static`` folder, and as an end result, we should have a project structure like the one :ref:`shown below <acos-exercise-folder-structure>`.
```

Comment:
How important is that the name of those files should not contain hyphens? Is it emphasized earlier? If not, should "the name of those files should not contain hyphens" be bolded or italicized or emphasized somehow?

Reply:
That sounds good, it would draw enough attention that one would notice that sentence. I wasn't sure if it was mentioned earlier, I meant to ask if it has been said earlier and emphasized then, as "Remember, --" gave me the impression that it might have been said earlier. If it had been said earlier and emphasized there, then it wouldn't need to be emphasized here (as it wouldn't be new and important information as it now is.)